src/wcm.cpp

```diff
@@ -338,28 +339,124 @@ OptionWidget::OptionWidget(Option *option) : Gtk::Box(Gtk::ORIENTATION_HORIZONTA
     switch (option->type)
     {
       case OPTION_TYPE_INT:
+      case OPTION_TYPE_ANIMATION:
     {
-        auto value_optional = wf::option_type::from_string<int>(
-            wf_option->get_value_str());
-        int value = value_optional ? value_optional.value() : std::get<int>(
-            option->default_value);
+        int value;
+        int default_value;
+        std::string easing_type;
```

Comment:
Maybe `default_easing_type` is a better name?

Reply:
`current_easing_type`, actually.